### ISSUE TITLE
The /var/www/html/ folder is completely empty in the latest 29 version of nextcloud

### ISSUE THREAD

**Issue by miltuss** [nextcloud/docker#2260]
The stable version 28.0.7 all configuration files are located in `/var/www/html`, like:

```
    /var/www/html/config
    /var/www/html/data
    /var/www/html/custom_apps
    /var/www/html/apps
    ...
```

Whereas in version 29 (latest), the configuration folders are completely disorganized. The folders `/var/www/html/` is completely empty. This means that the official nextcloud stack is totally wrong, because it keeps pointing to the `/var/www/html` folder which is completely empty in nextcloud:latest
https://github.com/nextcloud/docker?tab=readme-ov-file#base-version---apache

Furthermore the nextcloud files are located in another path which is not mentioned anywhere `/app/www/src/` !

![Sélection_404](https://github.com/user-attachments/assets/cc79af78-23a8-4834-b090-ff8646ef8145)

Why doesn't the last image store configuration files in `/var/www/html/`?

Why does the latest image compromise the version 28 update process?

**Comment by joshtrichards** [comment]
> Furthermore the nextcloud files are located in another path which is not mentioned anywhere /app/www/src/ !

Are you certain you're using this image and not some other image containing Nextcloud?

**Comment by miltuss** [comment]
> Are you certain you're using this image and not some other image containing Nextcloud? Sounds a bit like you might be loading the Linuxserver.io image.

I'm 90% sure to load the official nextcloud image.  Here is my stack:

```
version: "3"
services:
  nextcloud-test:
    image: nextcloud:latest
    container_name: nextcloud-test
    restart: always
    depends_on:
      - nextcloud-test_db
      - nextcloud-test_valkey
    ports:
      - 127.0.0.1:10100:80
    volumes:
      - /data/nextcloud-test/config:/var/www/html/config
      - /data/nextcloud-test/data:/var/www/html/data
      - /data/nextcloud-test/custom_apps:/var/www/html/custom_apps
      - /data/nextcloud-test/html:/var/www/html
    environment:
      - PUID=1000
      - PGID=1000
      - MYSQL_DATABASE=nextcloud
      - MYSQL_USER=nextcloud
      - MYSQL_PASSWORD=vhdfjhvbhkjdfb
      - MYSQL_HOST=nextcloud-test_db
      - REDIS_HOST=nextcloud-test_valkey
      - NEXTCLOUD_TRUSTED_DOMAINS=domain.com
  nextcloud-test_db:
    image: docker.io/library/mariadb
    container_name: nextcloud-test_db
    restart: always
    command: --transaction-isolation=READ-COMMITTED --binlog-format=ROW
    volumes:
      - /data/nextcloud-test/nextclouddb:/var/lib/mysql
    environment:
      - PUID=1000
      - PGID=1000
      - MYSQL_RANDOM_ROOT_PASSWORD=true
      - MYSQL_PASSWORD=vhdfjhvbhkjdfb
      - MYSQL_DATABASE=nextcloud
      - MYSQL_USER=nextcloud
  nextcloud-test_valkey:
    image: docker.io/valkey/valkey:alpine
    container_name: nextcloud-test_valkey
    volumes:
      - /data/nextcloud-test/valkey:/data
networks: {}
```

What is the full url should use for the nextcloud:latest image?
This way I will be sure that it is the official image that is used.

**Comment by miltuss** [comment]
You are right, it was the LSIO image that was used.  when I thought the stack was going to download the official image.  I will never use the LSIO image again

**Comment by joshtrichards** [comment]
>    `image: nextcloud:latest`

Any idea how you were ending up with the LSIO image with that Compose file?

Docker Hub is always used if a registry isn't specified. Well, and the local cache...

Were you testing with a different Compose (or maybe had a locally built image based on LSIO with the same tag in your local cache - I believe that might be one way)?

**Comment by miltuss** [comment]
> > `image: nextcloud:latest`
> 
> Any idea how you were ending up with the LSIO image with that Compose file?

I was using the LSIO image with the tak latest, and I recently decided to switch all my nextcloud instances to the official nextcloud image.  So when I indicated only nextcloud:latest in the compose file, it went to look for the LSIO image which was already downloaded instead of downloading the official NC image.  I too thought he was going to get the official NC image but he didn't. 

 So I deleted the LSIO image and restarted docker compose so that it downloaded the official image
